src/components/ModelsView.vue

```diff
@@ -20,6 +20,13 @@
           })"
         />
       </div>
+      <diagram-filter-card
+        v-model:selected-tags="selectedTags"
```

Comment:
2 way data binding is real necessary ?
https://vuejs.org/guide/components/v-model.html#multiple-v-model-bindings